extension/data/modules/queuetools.js

```diff
@@ -556,6 +558,16 @@ function queuetools () {
                 case 'flair':
                     selector = ':has(.linkflairlabel)';
                     break;
+                case 'pointsgt': {
+                    const min = parseInt(prompt('points greater than:', ''));
+                    selector = t => things[t].querySelector('.score.unvoted').innerHTML.split(' ')[0] > min;
```

Comment:
Turns out the `innerHTML` here isn't guaranteed to contain an actual number, things can get shortened to e.g. "196k" (check /top for an example). However, the `title` attribute does always have a number. I also simplified the selector a bit with jQuery.

```suggestion
                    selector = (_, el) => $(el).find('.score.unvoted').attr('title') > min;
```